### ISSUE TITLE
Librerías

### ISSUE THREAD

**Issue by AmbrosioV** [IIC2523/2020-2-Tarea2#15]
Se puede usar time.Sleep() para ayudar a la coordinación inicial? Ya tengo una forma que funciona sin esto, pero spamea muchos mensajes. Con esto podría evitar esto.

**Comment by FranLucchini** [comment]
Mi única preocupación es que el time.Sleep() tenga que ser muy largo dependiendo de cuántos procesos queramos iniciar ya que de antemano, no puedo saber cuánto me puedo demorar. Cuánto demora la versión con spam de mensajes?

Igualmente, creo que sería mejor hacer la coordinación por mensajes.

**Comment by SarahEverke** [comment]
¿Puede ser que una vez que estén todos los procesos conectados, el usuario aprete enter para que inicie la ejecución de las instrucciones?

**Comment by FranLucchini** [comment]
@SarahEverke sí, eso suena como una muy buena idea. En cualquier caso, lo ideal sería especificar ese funcionamiento en el informe.
